models/experimental/functional_distilbert/README.md

```diff
@@ -0,0 +1,28 @@
+## DIstilbert Model
+
```

Comment:
Please clarify what platforms (single card GS E150, WH N150, WH N300, T3K etc) this model is meant work for.

Customers with various systems will come and expect us to tell them what does and does not work.

Reply:
Sure, will check it

Reply:
This doesn't seem done, still?

Reply:
Updated the platforms on which the model works.